### PR TITLE
 added a post save signal to auto create the relevant user profile during registration

### PR DESCRIPTION
**Pull request by lili2311**
## Added back the console emails for dev too for faster testing

### CODE REVIEW

File: pimpmycause/pimpmycause/templates/profiles/profile.html [hunk 1]
```diff
@@ -8,109 +8,100 @@
...
+                                {{ profile_update_form.city.errors }}
+                                <label for="city"> City </label>
+                                <input type="text" name="city" placeholder="London" value='{{ profile_update_form.city.value|default_if_none:"" }}' required>
 
-                                    </div>
-                                </div>
-                            </div>
-                            <div class="row">
-                                <h2>Contact Info &amp; Social</h2>
+                                {{ profile_update_form.country.errors }}
```
Comment (yanabar) [inline pimpmycause/pimpmycause/templates/profiles/profile.html]: maybe we could add a label for country too ?
well unless if you find it unnecessary ..

Comment (yanabar) [inline pimpmycause/pimpmycause/templates/profiles/profile.html]: does this need to be below the form.city.errors ? around L58-58

File: pimpmycause/pimpmycause/templates/profiles/profile.html [hunk 2]
```diff
@@ -8,109 +8,100 @@
...
+                                {{ profile_update_form.country }}
```
Comment (yanabar) [inline pimpmycause/pimpmycause/templates/profiles/profile.html]: and this one also -as mentioned above ?

Comment (lili2311) [inline pimpmycause/pimpmycause/templates/profiles/profile.html]: these come from django and have both a label and input. If you do view Inspect in the browser your should be able to see it. Some fields are overriden manually and some just plucked out as they are in Django if the default looked good enough.

Comment (yanabar) [inline pimpmycause/pimpmycause/templates/profiles/profile.html]: oh i see what you mean..

**Comment by yanabar** [comment]
👍

**Pull request merged**